Fiber_Drawing_Tower/Arduino_and_Python_code/GUI.py

```diff
@@ -0,0 +1,282 @@
+import tkinter as tk
+import serial
+from tkinter import ttk
+import serial
+
+
+def serial_ports():
+    """ Finds all the port in use and returns it as a list
+    """
+    
+    #Try every possible port 
+    ports = ['COM%s' % (i + 1) for i in range(10)]
+
+    result = []
+    for port in ports:
+        try:
+            s = serial.Serial(port)
```

Comment:
When it's not meant for the current pull request, I recommand opening a new issue instead of a comment please in order to avoid a conversation blocking the pull request merging, thanks!